File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        mapa = {}
        for num in nums:
            mapa[num] = num

        longest = 0
        for i in range(len(nums)):
            num = nums[i]
            current_longest = 0
            if (num - 1 ) not in mapa:
                while num in mapa:
                    current_longest +=1
                    num +=1
            if current_longest >= longest:
                longest = current_longest
        return longest

File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int length = nums.length;
        int[] ans = new int[length*2];
        int j = length;

        for(int i=0; i<length;i++){
            ans[i] = nums[i];
            ans[j] = nums[i];
            j++;
        }
        return ans;
    }
}